bindings/rascal/neighbourlist/structure_manager.py

```diff
@@ -44,15 +44,18 @@ def __init__(self, frames, nl_options, start=None, length=None):
             managers = StructureCollectionFactory(nl_options)
             try:
                 managers.add_structures(structures)
-            except:
-                print("""Neighbourlist of structures failed. trying
-                one at a time.""")
+            except Exception as e:
+                print("Neighbourlist of structures failed because: " + str(e) +
+                "\n Trying one at a time.")
                 ii = 0
                 for structure, manager in zip(structures, managers):
                     try:
                         manager.update(structure)
-                    except:
-                        print("Structure Rep computation {} failed".format(ii))
+                    except Exception as e:
+                        print(
+                        "Neighbourlist of structure"+
+                        " {} failed because: ".format(ii) + str(e))
+                    ii += 1
```

Comment:
I do not see why we need the 'one at a time'. If one structures is failing half way through, it is not checked if the other would have gotten through?
I would get rid of the one-at-a-time and just fail. In my opinion it is very likely that other structures will fail, too. So why not clean up the whole data and retry...

Reply:
I keep it because I think it is useful when wrapping the atoms is not enough to avoid failures. You can single out the structure and try to understand better what happens. In principle there should not be failures if the atoms are wrapped but who knows.

Reply:
I don't see why this version helps you more when wrapping atoms is not enough - that should be a different error message anyways. And if you really have _bad_ structures, you could try them one by one outside by not supplying a list but checking them in a python loop like

Pseudo-example
```
for structure in structures:
    rep = compute(structure)
```

If the output of the error message would list _all_ structures, which have failed, I can see that as a valuable information, but if the looping is done only until it fails and possible structures in line behind that also fail, I don't see the point. It is basically hoping that there is only one offending structure. If you have multiple offending ones and you fix them one after another, it feels to me that it is like banging your head on the until the wall gives in. I myself at least would go the example route above if I encountered that issue twice.

So my suggestion is: either getting rid of the looping or make it useful by going through all the input structures and list them as a list of _failed_ structures with indices afterwards. Something like "structures [12,34,312,2422] were not computed because of errors 'blabla bla'. Then you can also see the different errors.